examples/todo/README.md

```diff
@@ -1 +1,539 @@
+
+The code below demonstrates this:
+
+```jsx
+// ./client/components/App.js
+
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { withViewModel } from 'resolve-redux';
+
+import actions from '../actions';
+
+const App = ({ todos, createItem, toggleItem, removeItem, aggregateId }) => {
+    let newToDo;
+    return (
+        <div>
+            <h1>To Do</h1>
+            <ol>
+                {Object.keys(todos).map(id => (
+                    <li key={id}>
+                        <label>
+                            <input
+                                type="checkbox"
+                                checked={todos[id].checked}
+                                onChange={toggleItem.bind(null, aggregateId, { id })}
+                            />
+                            {todos[id].text}
+                        </label>
+                        <span onClick={removeItem.bind(null, aggregateId, { id })}>{' [x]'}</span>
+                    </li>
+                ))}
+            </ol>
+            <input type="text" ref={element => (newToDo = element)} />
+            <button
+                onClick={() => {
+                    createItem(aggregateId, {
+                        text: newToDo.value,
+                        id: Date.now()
+                    });
+                    newToDo.value = '';
+                }}
+            >
+                Add Item
+            </button>
+        </div>
+    );
+};
+
+const mapStateToProps = (state) => {
+    const aggregateId = 'root-id';
+    const viewModel = 'ToDoView';
+    return {
+        viewModel,
+        aggregateId,
+        todos: state[viewModel][aggregateId]
+    };
+};
+
+const mapDispatchToProps = dispatch => bindActionCreators(actions, dispatch);
+
+export default connect(mapStateToProps, mapDispatchToProps)(withViewModel(App));
+```
+
+As you see, we use the current date in milliseconds as a ToDo item's **id**. This can also be any unique data, for instance, GUID.
+
+# Auxiliary Code
+
+We also need some auxiliary code to make things work. Consider the last three statements of the **App.js** file. They describe the connection between the Redux, React and reSolve. The functions defined and used there perform the following tasks:
+
+* **mapStateToProps**: Uses the **Redux State** to provide the **React Props** (`todos`) to the `App` conponent.
+* **mapDispatchToProp**: Uses the `bindActionCreators` function to provide the **React Callback Props** (`createItem`, `toggleItem`, and `removeItem`) to the `App` conponent.
+* **withViewModel**: Uses the `viewModel` and `aggregateId` from `mapStateToProps` for binding the **React Component** (`App`) to the **reSolve View Model** (`ToDoView`).
```

Comment:
Since `resolve-redux` version 0.2.2, there is no auxiliary entity called `withViewModel`. Instead, `connect` function should be imported not from `react-redux` package, but directly from `resolve-redux` package, which encapsulates required functionality.
Circumstance remained the same, so `aggregateId` and `viewModel` parameters are still actual.
See `todo` example source:
- https://github.com/reimagined/resolve/blob/master/examples/todo/client/components/App.js#L2
- https://github.com/reimagined/resolve/blob/master/examples/todo/client/components/App.js#L56